### PR TITLE
Added support to Hirefire

### PR DESCRIPTION
**Pull request by danilodelponte**
Testing:
1- Run rack app

``` Shell
thin -p 3000 start 
```

2- Access http://localhost:3000/hirefire/development/info

3- Should return number of jobs in queue for specified worker: 
  [{"name":"worker","quantity":0}]

4- Run outpost console:

``` Shell
. ./env.sh
irb -r ./runner.rb
```

5- Create an envelope and post it to create a job:

``` Ruby
envelope = MailCannon::Envelope.create(
  from: 'test@mailcannon.com',
  to: [{email: 'lucasmartins@railsnapraia.com', name: 'Lucas Martins'}],
  subject: 'Test',
  mail: MailCannon::Mail.new(text: 'you will see this when no HTML reader is available', html: 'this should be an HTML'))
envelope.post!
```

6-  Access http://localhost:3000/hirefire/development/info again and check if it returns the updated number of jobs:
 [{"name":"worker","quantity":1}]

Rack app staging url:
http://rdstation-mailcannon-staging.herokuapp.com/hirefire/5f7dca0c0792933e8053834f96a9de9f8782cd9e/info

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +0,0 @@
-rbx-2.2.1
```
Comment (lucasmartins) [inline .ruby-version:1]: You can't just remove this!

File: Gemfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby "2.1.0", :engine => "rbx", :engine_version => "2.2.1"
+#ruby "2.1.0", :engine => "rbx", :engine_version => "2.2.1"
```
Comment (lucasmartins) [inline Gemfile]: And this either...

File: config/mailcannon.yml [hunk 3]
```diff
@@ -1,17 +1,21 @@
 # This file will hold any hard configurations MailCannon may need. Customizations included.
+# You might want to use <%=ENV['var']%> to read theses value from your environment variables.
```
Comment (lucasmartins) [inline config/mailcannon.yml:2]: This was overriden by some local development file, all variables should be loaded from the environment: `<%=ENV['auto_post']%>`.

Please revert back to the original file.

File: config/sidekiq.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
-production:
-  :concurrency: <%=ENV['concurrency']%>
```
Comment (lucasmartins) [inline config/sidekiq.yml:3]: Same thing here, somebody overridden the original file.

File: .ruby-version (hunk 1 above)
Comment (danilodelponte) [inline .ruby-version:1]: woops isso é resquício da branch errada que pegamos inicialmente

File: Gemfile (hunk 2 above)
Comment (danilodelponte) [inline Gemfile]: E isso também

**Comment by lucasmartins** [comment]
@danilodelponte If it's easy to track down the bad commits you can just use `git revert`

**Comment by lucasmartins** [comment]
@danilodelponte I guess we should do another staging test after these commits are reverted.

**Comment by danilodelponte** [comment]
I'm using git checkout 'wrong_commit_sha1~1' path/to/file

**Pull request merged**